python/plugins/processing/tests/testdata/qgis_algorithm_tests3.yaml

```diff
@@ -575,13 +575,14 @@ tests:
             __all__:
               precision: 4
 
-  - algorithm: qgis:randompointsinextent
+  - algorithm: native:randompointsinextent
     name: Random point in extent, don't check result
     params:
-      TARGET_CRS: EPSG:4326
+      CRS: EPSG:4326
```

Comment:
You'll need to revert the changes to parameter names here, that's what's causing the Travis failure

Reply:
Thank you for being so patient with me - I totally missed that parameter.